src/formatting_stack/formatters/newlines.clj

```diff
@@ -13,8 +17,24 @@
                                  (impl/so-fix-them [filename] :expected-newline-count expected-newline-count)))))
   nil)
 
-(speced/defn new [{:keys [^pos-int? expected-newline-count]
-                   :or   {expected-newline-count 1}}]
+(defn lint! [{:keys [expected-newline-count]} files]
+  (binding [*out* (StringWriter.)
```

Comment:
Because we also bind *err* in `lint!`. Iirc the lib we're using prints the warnings and info to both stderr as stdout when linting (not when formatting). 

I could intercept out and err in both cases though.